Formula/cp2k.rb

```diff
@@ -30,10 +36,13 @@ class Cp2k < Formula
   end
 
   def install
+    # Issue with parallel build on macOS: https://github.com/cp2k/cp2k/issues/1316.
+    ENV.deparallelize if OS.mac?
```

Comment:
It seems weird to need this now, when we didn't need it before and neither the source nor the build configuration has changed.

Reply:
We were already using it on macOS but the way it was written before made it hard to make it macOS-only.  The cause of the parallel build failure is related to macOS lacking case sensitivity, so it is not needed on Linux.

Reply:
Yes, but previously we only needed it for some targets. Now we seem to be doing it everywhere.